server/routes/api.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express')
+const router = express.Router()
+const TransactionModel = require('../model/TransactionModel')
+
+router.get('/transactions', async function(req, res) {
+    try {
+        const data = await TransactionModel.find({}).sort('-date')
+        res.send(data)
```

Comment:
look into HTTP Status codes. You should get used to using them since you are already doing error handling